sdk/core/azure-core/azure/core/pipeline/transport/base.py

```diff
@@ -336,6 +336,20 @@ def _format_url_section(template, **kwargs):
                 components = [c for c in formatted_components if "{{{}}}".format(key.args[0]) not in c]
                 template = "/".join(components)
         # No URL sections left - returning None
+        
+    @staticmethod
+    def _urljoin(base_url, stub_url):
+        # type: (str, str) -> str
+        """Append to end of base URL without losing query parameters.
+        
+        :param str base_url: The base URL.
+        :param str stub_url: Section to append to the end of the URL path.
+        :returns: The updated URL.
+        :rtype: str
+        """
+        parsed = urlparse(base_url)
+        parsed = parsed._replace(path=parsed.path + '/' + stub_url)
```

Comment:
The parsed url has '/' at the end so the url is broken

Reply:
Fixed :)